bounded-collections/src/lib.rs

```diff
@@ -41,15 +41,33 @@ pub trait Get<T> {
 	fn get() -> T;
 }
 
-impl<T: Default> Get<T> for () {
+/// A trait for querying a single value from a type.
+///
+/// The value should be a constant.
+pub trait ConstGet<T> {
+	/// Return the constant value.
+	fn get() -> T;
+}
+
+/// Implement Get for any type that implements ConstGet.
```

Comment:
```suggestion
// Implement Get for any type that implements ConstGet.
```

impl blocks don't have rust-docs I think and should be documented with just `//`.

Reply:
They can have docs. They also appear in the docs.

Reply:
Used supertrait as suggested by @KiChjang.